marketplace/deployer_helm_tiller_base/marketplace-k8s-app-tools-helm-hooks/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+appVersion: 0.7.1
+description: Marketplace Helm integration
+home: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/
+keywords:
+- marketplace
+- helm
+maintainers:
+- email: cloud-marketplace-eng@google.copm
```

Comment:
not a public email alias. We probably have to set up one.

Reply:
Not really sure what the right long term solution is, but I'll start with a small list (trironkk@google.com). If it gets noisy I'll change it.